onnxruntime/core/providers/op_kernel_type_control.h

```diff
@@ -62,14 +62,14 @@ struct GlobalAllowed {};
 }  // namespace tags
 
 // optionally holds a list of types associated with a tag class
-// if types are defined, the data member 'types' should contain them in a type list
-// otherwise, if no types are defined (distinct from an empty list of types), there should be no data member 'types'
+// if types are defined, the type alias member called 'types' should contain them in a boost mp11 list
```

Comment:
>boost mp11 list [](start = 87, length = 15)

i think "type list/set" is more meaningful than "boost mp11 list/set".
if using "mp11 list", it would be good to copy or refer to the definition at [https://www.boost.org/doc/libs/1_75_0/libs/mp11/doc/html/mp11.html#definitions](https://www.boost.org/doc/libs/1_75_0/libs/mp11/doc/html/mp11.html#definitions)